c_fast.c

```diff
@@ -171,9 +170,8 @@ int main() {
     result = get_max_cost_small(costs, 0, 0);
   }
   gettimeofday(&end, NULL);
-  
+
   timersub(&end, &start, &duration);
   ms = duration.tv_sec*1000 + duration.tv_usec/1000;
-  printf("%d LANGUAGE C-fast %llu\n", result, ms);
+  printf("%d LANGUAGE C-fast %lu\n", result, ms);
```

Comment:
Sorry if that was not clear. `ms` has type `uint64_t`, which is why it has to be printed with `%llu` (which is a 64-bit unsigned integer both on 32-bit and 64-bit machines), and not with `%lu` (which is a 32-bit unsigned integer on (some) 32-bit machines and 64-bit on 64-bit machines).